packages/common/src/runtimes/DenoRuntime.ts

```diff
@@ -1,7 +1,9 @@
 // @ts-ignore
-import * as stdFs from "https://deno.land/std@0.89.0/fs/mod.ts";
+import { ensureDir, ensureDirSync } from "https://deno.land/std@0.104.0/fs/expand_glob.ts";
 // @ts-ignore
-import * as stdPath from "https://deno.land/std@0.89.0/path/mod.ts";
+import { expandGlob, expandGlobSync } from "https://deno.land/std@0.104.0/fs/ensure_dir.ts";
```

Comment:
Oops. The module specifiers are backwards.

Reply:
Fixed.